pvlib/singlediode.py

```diff
@@ -176,7 +217,9 @@ def bishop88(diode_voltage, photocurrent, saturation_current,
 
 def bishop88_i_from_v(voltage, photocurrent, saturation_current,
                       resistance_series, resistance_shunt, nNsVth,
-                      d2mutau=0, NsVbi=np.Inf, method='newton'):
+                      d2mutau=0, NsVbi=np.Inf, breakdown_factor=0.,
+                      breakdown_voltage=-5.5, breakdown_exp=3.28,
+                      method='newton'):
```

Comment:
As a convention, is it better to add new kwargs at the end, E.g. after `method='newton'`, so it doesn't break code that relies on position than explicit `kwarg=` usage? 

I'm half asking if that's something that's been historically done.

Reply:
I agree, but would like to keep like arguments grouped together in general order of decreasing interest: required module parameters first, then optional thin-film parameters, reverse bias parameters third, numerical control last.

Reply:
I think it's bad form to rely on the position of a keyword argument. Don't worry about it.